terraform/deployments/govuk-publishing-infrastructure/security.tf

```diff
@@ -207,3 +207,23 @@ resource "aws_security_group_rule" "licensify_frontend_from_eks_workers" {
   security_group_id        = data.terraform_remote_state.infra_security_groups.outputs.sg_licensify-frontend_internal_lb_id
   source_security_group_id = data.terraform_remote_state.cluster_infrastructure.outputs.node_security_group_id
 }
+
+resource "aws_security_group_rule" "clamav_efs_to_any_any" {
+  description       = "Clamav sends requests to anywhere over any protocol"
+  type              = "egress"
+  from_port         = 0
+  to_port           = 0
+  protocol          = -1
+  cidr_blocks       = ["0.0.0.0/0"]
+  security_group_id = aws_security_group.clamav-efs.id
+}
```

Comment:
I strongly suspect we don't need this rule at all - doesn't the client always initiate the TCP connection to the NFS server?

Reply:
Yeah, it is stateful, could remove that. Thanks.